src/components/contact-datalist-item/ContactDatalistItem.js

```diff
@@ -6,8 +6,9 @@ import DatalistItem from '../datalist-item';
 import './ContactDatalistItem.scss';
 
 type Props = {
-    name: React.Node,
+    name: string | null,
     subtitle?: React.Node,
+    title?: string | null,
```

Comment:
why is this needed?

Reply:
Not needed. It'll be necessary in the child component but not in this file